Loading pytorch-gpu/py3/1.6.0
  Loading requirement: gcc/8.3.1 cuda/10.2 nccl/2.6.4-1-cuda
    cudnn/7.6.5.32-cuda-10.2 intel-mkl/2020.1 magma/2.5.3-cuda
    openmpi/4.0.2-cuda
+ python3 -u run_resutls.py
Traceback (most recent call last):
  File "run_resutls.py", line 867, in <module>
    is_encoder_struct=True)
  File "run_resutls.py", line 188, in run_exp_extraction_and_visualization_custom_BK
    net_type=net_type, diff_var_loss=diff_var, z_struct_size=z_struct_size)
  File "run_resutls.py", line 461, in run_viz_expes
    same_binary_code(net, model_name, loader, nb_class, train_test=train_test, save=True, Hmg_dist=True)
  File "/gpfsdswork/projects/rech/rjg/uiu23ko/Thesis/Pytorch_vanilla_Beta_VAE/visualizer_CNN.py", line 2963, in same_binary_code
    _, embedding, _, _, _, _, _, _, _, _ = net(data, z_struct_out=True, z_struct_layer_num=z_struct_layer_num)
ValueError: too many values to unpack (expected 10)
+ wait
+ echo 'All done !'
